source/sass/donate/donate-main.scss

```diff
@@ -12,6 +12,9 @@
 // Non-React Components
 @import "../components/primary-nav";
 
+// Site-wide
```

Comment:
Without this import, the EOY donate banner was rendering like so:
![Screenshot 2024-12-11 at 18-21-27 Donate Now - Mozilla Foundation](https://github.com/user-attachments/assets/1fe41df3-59b6-4157-acdc-16861272291f)